common/gpu/intel/default.nix

```diff
@@ -37,7 +37,6 @@
         else
           intel-vaapi-driver
       )
-      libvdpau-va-gl
```

Comment:
This is also in line 52/51. Is that kept by accident?

Reply:
Yes. Good catch.